workflows/rnaseq/rnaseq-standard-fastq.wdl

```diff
@@ -29,110 +29,90 @@
 version 1.0
 
 import "../../tools/fq.wdl"
-import "../../tools/star.wdl"
-import "../../tools/picard.wdl"
-import "../../tools/ngsderive.wdl"
-import "../../tools/htseq.wdl"
-import "../../tools/samtools.wdl"
-import "../../tools/deeptools.wdl"
-import "https://raw.githubusercontent.com/stjude/XenoCP/3.1.4/wdl/workflows/xenocp.wdl" as xenocp_workflow
-import "../../tools/md5sum.wdl"
+import "./rnaseq-core.wdl" as rna_core
 
 workflow rnaseq_standard_fastq {
     input {
-        File gtf
         Array[File] read_one_fastqs
         Array[File] read_two_fastqs
-        File stardb_tar_gz
-        String strandedness = ""
-        String output_prefix
         String read_groups
-        Int max_retries = 1
-        Boolean detect_nproc = false
-        Boolean validate_input = true
+        String output_prefix
+        File gtf
+        File stardb
+        File? contaminant_stardb
```

Comment:
One minor point, we should be generic with `contaminant_db` and expose the `aligner` parameter to the user.

Reply:
Would a user ever want to use one of the `bwa` algorithms for XenoCP after using `star` in the main workflow? I figured a user would always want to be consistent with the aligner used.

Reply:
I suppose that's fair. I think it's OK to take an opinionated approach in our workflow and always use STAR. That said in our prior testing, there wasn't a clear advantage to using `STAR` over `bwa` for RNA-Seq data in this use case. STAR did unmap more reads, but not significantly so. It also had very little, if any, impact on the expression analysis.